### PR TITLE
tweak follow symlinks options for consistency

### PR DESCRIPTION
**Pull request by xabbuh**
@Tobion left some good comments in #828. This PR applies his suggestions.

### CODE REVIEW

File: web/.htaccess [hunk 1]
```diff
@@ -5,9 +5,10 @@
 # to each configured DirectoryIndex file (e.g. index.php, index.html, index.pl).
 DirectoryIndex app.php
 
-# uncomment the following line if you install assets as symlinks
-# or run into problems when compiling LESS/Sass/CoffeScript assets
-#
+# By default, Apache does not evaluate symbolic links if you did not enable this
+# feature in your server configuration. Uncomment the following line if you
+# install assets as symlinks or if experience problems related to symlinks when
```
Comment (fabpot) [inline web/.htaccess]: `if you experience`?

Comment (xabbuh) [inline web/.htaccess]: good catch, thanks

**Comment by weaverryan** [comment]
:+1: @fabpot would this library fall into the "2 core +1's => anyone can merge" category?

**Comment by fabpot** [comment]
@weaverryan Of course. Don't be shy :)

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**